client/core/core.go

```diff
@@ -5349,8 +5354,12 @@ func (c *Core) resumeTrades(dc *dexConnection, trackers []*trackedTrade) assetMa
 		tracker.recalcFilled()
 
 		if isActive {
-			tracker.lockRedemptionFraction(trade.Remaining(), trade.Quantity)
-			tracker.lockRefundFraction(trade.Remaining(), trade.Quantity)
```

Comment:
This was incorrect right? We were locking per match and here at the end. We should just do the targeted locks...

Reply:
Oh wait, this is locking unmatched funds.